src/loader.js

```diff
@@ -11,6 +11,36 @@ const NS = path.dirname(fs.realpathSync(__filename));
 
 export default source => source;
 
+function hmrCode() {
+  return (`
+if (module.hot) {
+  var injectCss = function injectCss(prev, href) {
+    var link = prev.cloneNode();
+    link.href = href;
+    link.onload = function() {
+      prev.parentNode.removeChild(prev);
+    };
+    prev.stale = true;
+    prev.parentNode.insertBefore(link, prev);
+  };
+  module.hot.dispose(function() {
+    window.__webpack_reload_css__ = true;
+  });
+  if (window.__webpack_reload_css__) {
+    module.hot.__webpack_reload_css__ = false;
+    console.log("[HMR] Reloading stylesheets...");
+    var prefix = document.location.protocol + '//' + document.location.host;
+    document
+      .querySelectorAll("link[href][rel=stylesheet]")
+      .forEach(function(link) {
+        if (!link.href.match(prefix) || link.stale) return;
+        injectCss(link, link.href.split("?")[0] + "?unix=${+new Date()}");
```

Comment:
```js
"?unix=${+new Date()}"
```

What's the purpose for this query param ? Is it needed ?

Reply:
yes, this is needed so hmr recognizes extracted stylesheet as modified and performs reload